### PR TITLE
Update Plugin Compatibility Tester from 0.2.1 to 0.3.0

### PR DESCRIPTION
**Pull request by oleg-nenashev**
https://github.com/jenkinsci/plugin-compat-tester/releases/tag/plugins-compat-tester-aggregator-0.3.0

### CODE REVIEW

File: prep.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ do
 done
 
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
-version=0.2.1
+version=0.3.0
 timestamp=20190926.172941-2 # TODO https://github.com/jenkinsci/plugin-compat-tester/pull/198
 pct=$HOME/.m2/repository/org/jenkins-ci/tests/plugins-compat-tester-cli/${version}-SNAPSHOT/plugins-compat-tester-cli-${version}-${timestamp}.jar
```
Comment (jglick) [inline prep.sh]: Need to revert the `timestamp` stuff added in 453f3d3b4d4444c77cfa62f42f7636fad1883a16 to pick up https://github.com/jenkinsci/plugin-compat-tester/pull/198 which I waited for this release.

Comment (oleg-nenashev) [reply prep.sh]: Just removed it. Of a more robust implementation, there might be a follow-up

File: prep.sh [hunk 2]
```diff
@@ -40,9 +40,8 @@ do
 done
 
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
-version=0.2.1
-timestamp=20190926.172941-2 # TODO https://github.com/jenkinsci/plugin-compat-tester/pull/198
-pct=$HOME/.m2/repository/org/jenkins-ci/tests/plugins-compat-tester-cli/${version}-SNAPSHOT/plugins-compat-tester-cli-${version}-${timestamp}.jar
+version=0.3.0
+pct=$HOME/.m2/repository/org/jenkins-ci/tests/plugins-compat-tester-cli/${version}-SNAPSHOT/plugins-compat-tester-cli-${version}.jar
```
Comment (jglick) [inline prep.sh]:
```suggestion
pct=$HOME/.m2/repository/org/jenkins-ci/tests/plugins-compat-tester-cli/${version}/plugins-compat-tester-cli-${version}.jar
```

File: prep.sh [hunk 3]
```diff
@@ -40,9 +40,8 @@ do
 done
 
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
-version=0.2.1
-timestamp=20190926.172941-2 # TODO https://github.com/jenkinsci/plugin-compat-tester/pull/198
-pct=$HOME/.m2/repository/org/jenkins-ci/tests/plugins-compat-tester-cli/${version}-SNAPSHOT/plugins-compat-tester-cli-${version}-${timestamp}.jar
+version=0.3.0
+pct=$HOME/.m2/repository/org/jenkins-ci/tests/plugins-compat-tester-cli/${version}/plugins-compat-tester-cli-${version}.jar
 [ -f $pct ] || $MVN dependency:get -Dartifact=org.jenkins-ci.tests:plugins-compat-tester-cli:${version}-${timestamp}:jar -DremoteRepositories=https://repo.jenkins-ci.org/public/ -Dtransitive=false
```
Comment (jglick) [inline prep.sh]:
```suggestion
[ -f $pct ] || $MVN dependency:get -Dartifact=org.jenkins-ci.tests:plugins-compat-tester-cli:${version}:jar -DremoteRepositories=https://repo.jenkins-ci.org/public/ -Dtransitive=false
```

**Comment by jglick** [comment]
> there might be a follow-up

You could just set up Incrementals. It is already using the parent POM and ought to be in RPU, so should be easy enough using `infra.prepareToPublishIncrementals` / `.maybePublishIncrementals`.

**Comment by oleg-nenashev** [comment]
Sorry, I should not edit PRs from phone

**Comment by basil** [comment]
Unfortunately, I do not think PCT 0.3.0 was released correctly. I have empirically observed that it does not contain my fix for jenkinsci/plugin-compat-tester#199 as mentioned in [the release notes](https://github.com/jenkinsci/plugin-compat-tester/releases/tag/plugins-compat-tester-aggregator-0.3.0). From `git log`:

```
$ git log -5 plugins-compat-tester-aggregator-0.3.0
commit 20d334349a296a68dd152cd3e9b76b7ad99cc558 (tag: plugins-compat-tester-aggregator-0.3.0)
Author: Oleg Nenashev <o.v.nenashev@gmail.com>
Date:   Fri Oct 4 08:38:56 2019

    [maven-release-plugin] prepare release plugins-compat-tester-aggregator-0.3.0

commit ea23cef4255ffab19f63de2d4d5a5e4916adfd6b
Merge: 4ad3bfc ead8247
Author: Oleg Nenashev <o.v.nenashev@gmail.com>
Date:   Thu Oct 3 03:20:24 2019

    Merge branch 'JENKINS-58113-remove-maven-internal' of https://github.com/oleg-nenashev/plugin-compat-tester into JENKINS-58113-remove-maven-internal

commit 4ad3bfc114301e9212c82292a983fababc74892a
Author: Oleg Nenashev <o.v.nenashev@gmail.com>
Date:   Thu Oct 3 03:19:18 2019

    [JENKINS-58113] - Remove more dependencies after GAE removal

commit 0233e061f5e603aa1345663b3b445aaab3f7262d
Merge: 6781cc4 ba980f3
Author: Oleg Nenashev <o.v.nenashev@gmail.com>
Date:   Thu Oct 3 01:33:56 2019

    Merge remote-tracking branch 'jenkinsci/master' into JENKINS-58113-remove-maven-internal

commit ba980f3b03eb44fd4ea8faf9d88ee839b87e461e
Merge: f7d00ce 1d8d9da
Author: Raúl Arabaolaza Barquin <rarabaolaza@cloudbees.com>
Date:   Mon Sep 30 02:33:49 2019

    Merge pull request #192 from basil/testScoped
    
    Plugin compatibility tester is re-adding test-scoped dependencies rather than merely updating them
```

Looks like PCT was erroneously released from a local branch corresponding to unmerged PR jenkinsci/plugin-compat-tester#194 rather than `master`.

**Comment by basil** [comment]
> Looks like PCT was erroneously released from a local branch corresponding to unmerged PR [jenkinsci/plugin-compat-tester#194](https://github.com/jenkinsci/plugin-compat-tester/pull/194) rather than `master`.

@oleg-nenashev Gentle ping. Would you be willing to re-release PCT from the correct branch?

**Pull request merged**